core/src/main/scala/org/apache/spark/api/java/JavaSparkContext.scala

```diff
@@ -238,7 +238,9 @@ class JavaSparkContext(val sc: SparkContext)
    * }}}
    *
    * Do
-   * `JavaPairRDD<String, byte[]> rdd = sparkContext.dataStreamFiles("hdfs://a-hdfs-path")`,
+   * <code>
+   *   JavaPairRDD&lt;String, byte[]&gt; rdd = sparkContext.dataStreamFiles("hdfs://a-hdfs-path")
+   * </code>,
```

Comment:
note to myself. It still prints the codes as above.

If we want to use `<` or `>`, we might better always wrap this with

```
{{{
...
}}}
```

rather than backticks or `<code>...</code>`.